docs/mp/oci/01_oci.adoc

```diff
@@ -57,7 +57,7 @@ credentials from your environment.
 
 When you inject an OCI SDK Client object, the Helidon OCI SDK extension
 configures and constructs the object for you. The configuration primarily
-consists of initializing an OCI Authentication Details Provider. By default
+consists of initializing an OCI `AuthenticationDetailsProvider`. By default
 the extension will examine your environment and select the best Authentication
```

Comment:
(Technically you should change this reference too; up to you.)

Reply:
Fixed